api/envoy/service/auth/v2alpha/external_auth.proto

```diff
@@ -24,21 +24,24 @@ message CheckRequest {
   AttributeContext attributes = 1;
 }
 
+// An message that contains HTTP response attributes. This message is
+// used when the authorization service needs to send custom responses to the
+// downstream client or, to modify/add request headers being dispatched to the upstream.
+message HttpResponse {
+  // Http status code.
+  uint32 status_code = 1 [(validate.rules).uint32 = {gte: 100, lt: 600}];
```

Comment:
Non-actionable, but if we were to redo the APIs from the ground up, I think I would add a dedicated `HttpStatusCode` object with the related constraints. We've done this now in a lot of places.

Reply:
Sounds good. Should I place it here in a separated file like the `http_uri.proto`? https://github.com/gsagula/envoy/tree/e6d6df09c84a189a43762a1e6839103e8d02c51e/api/envoy/api/v2/core